Marlin/src/gcode/control/M605.cpp

```diff
@@ -147,16 +147,20 @@
     #endif // DEBUG_DXC_MODE
   }
 
-#elif ENABLED(DUAL_NOZZLE_DUPLICATION_MODE)
+#elif ENABLED(MULTI_NOZZLE_DUPLICATION_MODE)
 
   void GcodeSuite::M605() {
-    planner.synchronize();
-    extruder_duplication_enabled = parser.intval('S') == (int)DXC_DUPLICATION_MODE;
-    SERIAL_ECHO_START();
-    SERIAL_ECHOPGM(MSG_DUPLICATION_MODE);
-    serialprintln_onoff(extruder_duplication_enabled);
+    if(parser.intval('S')) {}
+      planner.synchronize();
+      extruder_duplication_enabled = (int)DXC_DUPLICATION_MODE;
```

Comment:
A few issues with these lines:
- `extruder_duplication_enabled` is a `bool`.
- The 'S' value is not being used here, except non-zero enters the block (or will, once the typo is fixed).
- In order for `M605 S` to act as an enable, you must use `boolval('S')`.
- In order for `M605 S0` to act as a disable, you must set `extruder_duplication_enabled` to `false`.

Reply:
Yeah, when I came up with the question of how we want to handle arguments I just dropped it there, pushed for feedback, and moved on for the moment. Need that question resolved before I wrap it up since it'll replace this code block completely.

Reply:
If feedback is slow in coming, you may find one or two examples of what you're trying to do in the sources.